src/test/groovy/edu/ie3/datamodel/utils/validation/NodeValidationUtilsTest.groovy

```diff
@@ -27,15 +27,15 @@ class NodeValidationUtilsTest extends Specification {
     def node = GridTestData.nodeA
 
     when:
-    NodeValidationUtils.check(node)
+    List<Try<Void, ? extends ValidationException>> tries = NodeValidationUtils.check(node)
 
     then:
-    noExceptionThrown()
+    tries.forEach { it.success }
```

Comment:
Use `every` here, otherwise it won't fail. I also found a bunch of tests from the previous PR using `forEach`, please fix them as well.